### DISCUSSION TITLE
Failed building wheel for insightface

### DISCUSSION THREAD

**Discussion by K4ndiszucker** [hacksider/Deep-Live-Cam#195] [Q&A] [answered]
this error comes up when I install the requirements.txt 
everything else worked, 

Building wheel for insightface (pyproject.toml) ... error
  error: subprocess-exited-with-error
  × Building wheel for insightface (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [230 lines of output]

  ERROR: Failed building wheel for insightface
Failed to build insightface
ERROR: Could not build wheels for insightface, which is required to install pyproject.toml-based projects

**Comment by vlednerk** [comment, marked as answer]
```
> winget install -e --id Microsoft.VCRedist.2015+.x64
> winget install -e --id Microsoft.VisualStudio.2022.BuildTools --override "--wait --add Microsoft.VisualStudio.Workload.NativeDesktop --includeRecommended"
```
![270335592-c77ccdf9-7f63-4186-9703-f9d28ad61bdb](https://github.com/user-attachments/assets/fca84f00-9bc3-4df7-be3e-80e46d46e6e9)

**Reply by K4ndiszucker** [reply]
thanks
